README.md

```diff
@@ -370,6 +372,32 @@ The documented advanced deployment scenarios may introduce additional custom env
 | ---------------------------------------------- | ------------------------------------------------------------- |
 | `CORTX_DEPLOY_CUSTOM_BLOCK_STORAGE_CLASS`      | When set to a non-empty string, CORTX will skip the deployment of the `cortx-block-data` Helm Chart, along with the automatic creation of the underlying PersistentVolumes necessary to deploy CORTX. It will instead use the value of this variable as the expected Kubernetes StorageClass and expect all available PersistentVolumes to be created manually by the user. See [Advanced Deployment Scenarios - Using manually-created PersistentVolumes](doc/advanced-deployment-scenarios.md#using-manually-created-persistentvolumes) for use case details.  |
 
+### Deploy with SSL Certificate
+
+By default, CORTX is installed with a self-signed SSL certificate.  This section describes how to deploy with a custom SSL instead.
+
+Before deploying CORTX, create a Kubernetes Secret that contains the SSL certificate.
+
+  * The Secret must contain the key `cortx.pem`
+  * The value must be in PEM format and must contain the Private Key and Certificate.
+
+   ```bash
+   # where "mycert.pem" contains the certificate
+   kubectl create secret generic my-ssl-cert --from-file=cortx.pem=mycert.pem -n "${NAMESPACE}"
+   ```
+
+Specify common.ssl.external_secret in `solution.yaml`:
+
+   ```yaml
+   solution:
+     common:
+       ssl:
+         external_secret: my-ssl-cert
+   ```
+
+When you deploy CORTX, it will now use the SSL certificate from the Secret `my-ssl-cert` instead of the efault self-signed SSL certificate.
```

Comment:
```suggestion
When you deploy CORTX, it will now use the SSL certificate from the Secret `my-ssl-cert` instead of the default self-signed SSL certificate.
```